assets/sass/patterns/organisms/content-header-journal.scss

```diff
@@ -326,10 +326,6 @@
 
     .divider {
       display: none;
-
-      @media only all and (min-width: #{get-rem-from-px($bkpt-site--extended-wide)}em) {
-        display: block;
-      }
```

Comment:
The style for divider will apply only in "content-header__has-aside" class, which is used only in pages with Content header and Aside component, so it won't affect other pages.

We don't need to hide/show the sections below, tab click should trigger page reload, as it is on [EPP](https://elifesciences.org/reviewed-preprints/84918).
@chris-huggins, please, correct me if I'm wrong.

Reply:
You seem to confirm these are breaking changes. This is probably an example of a change that we need to see working in journal before we merge it into pattern-library. It may not affect other pages but it will affect article pages with a content aside unless they have this new component. This component is not ready. The js disables the behaviour of the links so they will not be followed. This was presumably to demonstrate the intended behaviour but it prevents it from following the links.